diff --git a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
index 1848ae8..4bd220f 100644
--- a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
+++ b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "10.0" }
+  s.platforms    = { :ios => "15.1" }
   s.source       = { :git => "https://github.com/taboulot/react-native-image-resizer.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm}"
@@ -27,7 +27,7 @@ Pod::Spec.new do |s|
         "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
     }
     s.dependency "React-Codegen"
-    s.dependency "RCT-Folly"
+    # s.dependency "RCT-Folly"
     s.dependency "RCTRequired"
     s.dependency "RCTTypeSafety"
     s.dependency "ReactCommon/turbomodule/core"
