diff --git a/node_modules/@react-native-community/blur/package.json b/node_modules/@react-native-community/blur/package.json
index 4e5c8a6..e3ebe8d 100644
--- a/node_modules/@react-native-community/blur/package.json
+++ b/node_modules/@react-native-community/blur/package.json
@@ -2,9 +2,9 @@
   "name": "@react-native-community/blur",
   "version": "4.4.1",
   "description": "React Native Blur component",
-  "main": "lib/commonjs/index",
-  "module": "lib/module/index",
-  "types": "lib/typescript/index.d.ts",
+  "main": "src/index",
+  "module": "src/index",
+  "types": "src/index",
   "react-native": "src/index",
   "source": "src/index",
   "files": [
diff --git a/node_modules/@react-native-community/blur/src/components/BlurView.web.tsx b/node_modules/@react-native-community/blur/src/components/BlurView.web.tsx
new file mode 100644
index 0000000..b2a9b40
--- /dev/null
+++ b/node_modules/@react-native-community/blur/src/components/BlurView.web.tsx
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View } from 'react-native-web';
+
+// defaults from native library
+const DEFAULT_BLUR_AMOUNT = 10;
+const DEFAULT_BLUR_TYPE = 'dark';
+
+const getBackgroundColor = (blurType) => {
+  switch (blurType) {
+    case 'light':
+      return 'rgba(255,255,255,0.5)';
+    case 'dark':
+      return 'rgba(0,0,0,0.5)';
+    default:
+      // will just blur without any tinting.
+      return 'transparent';
+  }
+};
+
+const BlurView = ({ blurType = DEFAULT_BLUR_TYPE, blurAmount = DEFAULT_BLUR_AMOUNT, style }) => (
+  <View style={[
+    {
+      backdropFilter: `blur(${blurAmount}px)`,
+      backgroundColor: getBackgroundColor(blurType),
+    },
+    style,
+  ]}
+  />
+);
+
+export { BlurView };
diff --git a/node_modules/@react-native-community/blur/src/components/VibrancyView.web.tsx b/node_modules/@react-native-community/blur/src/components/VibrancyView.web.tsx
new file mode 100644
index 0000000..9f95e66
--- /dev/null
+++ b/node_modules/@react-native-community/blur/src/components/VibrancyView.web.tsx
@@ -0,0 +1,10 @@
+import React from 'react';
+
+class VibrancyView extends React.Component {
+  render() {
+    console.error('VibrancyView is not implemented on Android');
+    return null;
+  }
+}
+
+export default VibrancyView;
