diff --git a/node_modules/react-native-scalable-image/index.js b/node_modules/react-native-scalable-image/index.js
index 655e3b3..a5f6d51 100644
--- a/node_modules/react-native-scalable-image/index.js
+++ b/node_modules/react-native-scalable-image/index.js
@@ -12,8 +12,6 @@ import {
     ImageBackground
 } from 'react-native';
 
-const resolveAssetSource = Image.resolveAssetSource;
-
 const ScalableImage = props => {
     const ImageComponent = props.component
         ? props.component
@@ -64,8 +62,11 @@ const ScalableImage = props => {
             );
         }
         else {
-            const sourceToUse = resolveAssetSource(source);
-            adjustSize(sourceToUse.width, sourceToUse.height, props);
+            Image.getSize(
+              source,
+              (width, height) => adjustSize(width, height, props),
+              console.err
+            );
         }
     };
 
