diff --git a/node_modules/react-google-maps/lib/withGoogleMap.js b/node_modules/react-google-maps/lib/withGoogleMap.js
index e445631..e8974a3 100644
--- a/node_modules/react-google-maps/lib/withGoogleMap.js
+++ b/node_modules/react-google-maps/lib/withGoogleMap.js
@@ -68,7 +68,7 @@ function _interopRequireDefault(obj) {
 
 /* global google */
 function withGoogleMap(BaseComponent) {
-  var factory = _react2.default.createFactory(BaseComponent)
+  // var factory = _react2.default.createFactory(BaseComponent)
 
   var Container = (function(_React$PureComponent) {
     ;(0, _inherits3.default)(Container, _React$PureComponent)
@@ -169,7 +169,7 @@ function withGoogleMap(BaseComponent) {
               _react2.default.cloneElement(mapElement, {
                 ref: this.handleComponentMount,
               }),
-              _react2.default.createElement("div", null, factory(restProps))
+              _react2.default.createElement("div", null, BaseComponent)
             )
           } else {
             return _react2.default.cloneElement(
diff --git a/node_modules/react-google-maps/lib/withScriptjs.js b/node_modules/react-google-maps/lib/withScriptjs.js
index 21084d0..68f16ef 100644
--- a/node_modules/react-google-maps/lib/withScriptjs.js
+++ b/node_modules/react-google-maps/lib/withScriptjs.js
@@ -65,7 +65,7 @@ var LOADING_STATE_BEGIN = "BEGIN"
 var LOADING_STATE_LOADED = "LOADED"
 
 function withScriptjs(BaseComponent) {
-  var factory = _react2.default.createFactory(BaseComponent)
+  // var factory = _react2.default.createFactory(BaseComponent)
 
   var Container = (function(_React$PureComponent) {
     ;(0, _inherits3.default)(Container, _React$PureComponent)
@@ -169,7 +169,7 @@ function withScriptjs(BaseComponent) {
           var loadingState = this.state.loadingState
 
           if (loadingState === LOADING_STATE_LOADED) {
-            return factory(restProps)
+            return BaseComponent
           } else {
             return loadingElement
           }
